src/app/workflow/workflow-file-editor/workflow-file-editor.component.ts

```diff
@@ -60,10 +62,12 @@ export class WorkflowFileEditorComponent extends FileEditing {
 
     this.hostedService.editHostedWorkflow(
         this.id,
-        newSourceFiles).subscribe(result => {
+        newSourceFiles).subscribe(workflow => {
           this.toggleEdit();
-          this.workflowService.setWorkflow(result);
           this.refreshService.handleSuccess(message);
+          this.workflowsService.getWorkflow(workflow.id).subscribe((workflow2) => {
```

Comment:
type, what does the workflow of this endpoint return that the editHostedWorkflow endpoint doesn't return?

Reply:
That is a good question

Reply:
Oh yes it was something to do with the editHostedWorkflows not returning the users array, but getWorkflow does.